google/auth/credentials_async.py

```diff
@@ -0,0 +1,347 @@
+# Copyright 2016 Google LLC
```

Comment:
Would also be helpful to see a diff of this file vs. credentials.py. (I don't think there's a super straightforward way to do this in git/github unfortunately unless you go back and make the initial commit just a copy of the file and then stack subsequent commits on that, but you could also just take a diff and send it to us separately).

Reply:
Resolved (refactored to remove credentials_async file - did not need async/await on refreshing credentials)

Reply:
Looks like this file is still present? Is it just because of before_request?